### ISSUE TITLE
Lines in game tree not drawing

### ISSUE THREAD

**Issue by fohristiwhirl** [SabakiHQ/Sabaki#549]
If I have a pathologically branchy SGF file (like what might be produced by analysis tools) such that no node appears in the tree visualiser (because the lines between nodes are so long) the lines themselves aren't drawn.

I've never seen this before and wonder if it's a new issue? I'm on the current source, I believe.

[example.sgf.txt](https://github.com/SabakiHQ/Sabaki/files/3076698/example.sgf.txt)

**Comment by dbosst** [comment]
Inspecting the component, the svg produced is indeed empty, so not a chrone-render glitch

**Comment by yishn** [comment]
Yes, @dbosst is correct. We only draw the edges of nodes that are visible in the UI. For edges, there's currently no easy way to determine if one is visible or not...
